src/os/bluestore/BlueStore.cc

```diff
@@ -1693,24 +1663,33 @@ BlueStore::BufferCacheShard *BlueStore::BufferCacheShard::create(
 #undef dout_prefix
 #define dout_prefix *_dout << "bluestore.BufferSpace(" << this << " in " << cache << ") "
 
-void BlueStore::BufferSpace::_clear(BufferCacheShard* cache)
+bool BlueStore::BufferSpace::_dup_writing(TransContext* txc, BufferCacheShard* cache, OnodeRef onode)
```

Comment:
Was changing a location (and dropping func description) of _dup_writing deliberate?
If not, its better to minimize diff.

Reply:
I will move that back